src/WinRT.Runtime/Projections/Nullable.cs

```diff
@@ -2364,6 +2390,8 @@ unsafe object IWinRTNullableTypeDetails.GetNullableValue(IInspectable inspectabl
                 Marshal.Release(nullablePtr);
             }
         }
+
+        Type IWinRTNullableTypeDetails.GetNullableType() => typeof(global::System.Nullable<T>);

```

Comment:
Nullable support is disabled when the `IReference<T>` feature switch is disabled, right? I'm wondering if we should leverage that and make this method just throw (or return `null`?) when that feature switch is disabled, to avoid the generic `Nullable<T>` instantiation for all these types.

Reply:
This is implemented as an internal interface and all uses of the internal interface are behind the feature check, so that should allow all of this to get trimmed away.  To confirm, I did some sizoscope comparisons, the interface implementations are all getting trimmed away, so there are no nullable instantiations.  As a result, I don't think we will need another feature check here.

Reply:
Ah, cool, that works too! Thank you for double checking 🙂